packages/vite/src/node/ssr/ssrExternal.ts

```diff
@@ -138,7 +138,7 @@ export function createIsConfiguredAsSsrExternal(
         // Return undefined here to avoid short-circuiting the isExternalizable check,
```

Comment:
Now that we're returning `true` here, does this comment still apply?

Reply:
Updated, thanks!